lnrpc/lightning.proto

```diff
@@ -2190,6 +2190,33 @@ message OpenChannelRequest {
     attempted.
     */
     bool scid_alias = 20;
+
+    /*
+    The base fee charged regardless of the number of milli-satoshis sent.
+    */
+    uint64 base_fee = 21;
+
+    /*
+    The fee rate in ppm (parts per million) that will be charged in
+    proportion of the value of each forwarded HTLC.
+    */
+    uint64 fee_rate = 22;
+
+    /*
+    If UseBaseFee is true the open channel announcement will update the
```

Comment:
We normally use the notation of the proto variables and not the resulting Golang names. So we should use `use_base_fee` and `base_fee` in the comments instead of `UseBaseFee`.